hw/ip_templates/pwrmgr/dv/sva/pwrmgr_unit_only_sva.core

```diff
@@ -2,8 +2,8 @@ CAPI=2:
 # Copyright lowRISC contributors (OpenTitan project).
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:dv:rstmgr_unit_only_sva:0.1"
-description: "RSTMGR assertion modules not suitable for chip level and bind file."
+name: "lowrisc:dv:pwrmgr_unit_only_sva:0.1"
+gdescription: "PWRMGR assertion interfaces not suitable for chip level bind file."
```

Comment:
**g**description is probably a typo?
```suggestion
description: "PWRMGR assertion interfaces not suitable for chip level bind file."
```

Reply:
This seems to cause the failure of the *in-depth lint* CI check

Reply:
Fixed, thanks.